### DISCUSSION TITLE
Strange new error

### DISCUSSION THREAD

**Discussion by douglas-ha** [rustdesk/rustdesk#7933] [General]
### Bug Description

When I try to connect to a computer overseas that I have done 100's of times I am ow getting a message 

Connection error The connection is not allowed. To further prevent scamming, we only allow connections between users in the same city.

Any idea what is causing this ?

### How to Reproduce

Every time I try to connect it does this and from 2 different computers

### Expected Behavior

I expect it to connect.

### Operating system(s) on local side and remote side

W11 on both ends

### RustDesk Version(s) on local side and remote side

Version: 1.2.3-2 on Client side, do not remember version on server side

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/66419347/ffd49425-fea3-4c8a-b9fb-eaec8dcd6708)

### Additional Context

![image](https://github.com/rustdesk/rustdesk/assets/66419347/703ab4e8-9dc3-468a-b295-91e8fe9f0709)

**Comment by ohm5555555555** [comment]
ahh me too how to fix :( 
![image](https://github.com/rustdesk/rustdesk/assets/169118210/222ea7b1-182b-4565-a150-230b6e4af247)

**Comment by Nephilim94** [comment]
I also facing the same issue. When I reinstall the application it get crashing within 5 seconds of start up.

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1787689958276472982?t=Cn1RQ7S6OpZsa8UfqohbNA&s=19
